web/pandas/about/team.md

```diff
@@ -36,6 +36,14 @@ If you want to support pandas development, you can find information in the [dona
     {% endfor %}
 </div>
 
+## Diversity and Inclusion
+ 
+> _pandas_ expressly welcomes and encourages anyone who faces under-representation, discrimination in the technology industry 
+> or anyone willing to increase the diversity of our team, we encourage you to contribute.
+> We have identified the visible diversity gap and obstacles in the open source community and we are proactive in increasing the diversity of our team.
+> We have a [code of conduct]({base_url}/community/coc.html) to ensure a friendly and welcoming environment. 
+> Please send us a message to [pandas team](pandas-coc@googlegroups.com), if you don't feel this way.
```

Comment:
the message won't be received by the pandas team, but by the pandas code of conduct committee, so better change the link text.

Can you also add `mailto:` before the email address, so it becomes a mail link.